build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-NAMESPACE="uofa"
+NAMESPACE=$(git remote get-url origin | awk -F'[/:]' '{ print $2 }')
```

Comment:
This just has to pass the "Does it work on a Mac?" test.

Reply:
`-F` flag option is available on mac. Running 
`git remote get-url origin | awk -F'[/:]' '{ print $2 }'` on this repo produces : `universityofadelaide` 

🎉  👍

Reply:
Yeah, now I know about multiple field delimeters in awk, I know a few places that will come in handy!